scripts/colab_install.py

```diff
@@ -15,29 +15,58 @@
 logger.setLevel(INFO)
 
 
-def install(chunk_size=4096,
-            file_name="Miniconda3-latest-Linux-x86_64.sh",
-            url_base="https://repo.continuum.io/miniconda/",
-            conda_path=os.path.expanduser(os.path.join("~", "miniconda")),
-            version=None,
-            gpu=True,
-            add_python_path=True,
-            force=False):
-  """install deepchem from miniconda on Google Colab
-
-    For GPU notebook
+default_channels = [
+    "-c",
+    "rdkit",
+    "-c",
+    "conda-forge",
+]
+default_packages = [
+    "rdkit",
+]
+
+
+def install(
```

Comment:
I refactored the more flexible script. I will make the update tutorials in another PR.